tests/test_impls.py

```diff
@@ -2,17 +2,28 @@
 
 import os.path
 import tempfile
+from contextlib import contextmanager
 from unittest.mock import Mock, patch
 
 import pytest
 
 from rapids_build_backend.impls import (
-    _VERSIONED_RAPIDS_WHEELS,
-    _add_cuda_suffix,
     _edit_git_commit,
+    _edit_pyproject,
+    _get_cuda_suffix,
 )
 
 
+@contextmanager
+def set_cwd(cwd):
```

Comment:
I really don't know why this doesn't exist in Python yet :( I've rewritten it so many times.